multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/UpdateServiceParametersStep.java

```diff
@@ -18,6 +20,8 @@
 @Scope(BeanDefinition.SCOPE_PROTOTYPE)
 public class UpdateServiceParametersStep extends ServiceStep {
 
+    private static final String PARAMETERS = "parameters";
```

Comment:
if you still need these parameters after fixing the above comment, move them to another class with constants/messages. 
Same applies for other tags, syslog, etc